.goreleaser.docker.yaml

```diff
@@ -280,6 +303,52 @@ dockers:
       - '--label=org.opencontainers.image.title=jvs-ui'
       - '--label=org.opencontainers.image.version={{ .Version }}'
 
+  -
+    ids:
+    - jvsctl
```

Comment:
Sorry, it just occurred to me this should be named as `jvs-prober`. We used the `jvsctl` binary but the container image is a prober.